pom.xml

```diff
@@ -5,7 +5,7 @@
 
 	<groupId>org.springframework.data</groupId>
 	<artifactId>spring-data-redis</artifactId>
-	<version>2.6.0-SNAPSHOT</version>
+	<version>2.6.0-2047-SNAPSHOT</version>
```

Comment:
We only change the version if we push code to one of our repository branches to avoid CI builds producing artifacts with the same version identifier. External pull requests do not require that change. We will skip it during the merge.